src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 300ms, transform 300ms;
```

тут color писати не потрібно.
```suggestion
    transition: transform 300ms;
```

---

@YuriiPronin why? 🥲